composer.json

```diff
@@ -19,7 +19,7 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
+        "php": ">=5.5.0",
```

Comment:
This is because `DateTimeInterface` was introduced in PHP 5.5. But the fix can be done without referencing `DateTimeInterface`, if you don't want to bump PHP version.

Reply:
It's fine. I will actually bump the code to 7.x. PHP 5.x is for a long time unsupported/maintained. Who still using old PHP can still use a previous version of the lib.